news/583.api_change

```diff
@@ -0,0 +1 @@
+`DictConfig.{get(),pop()}` and `ListConfig.get()` now return `None` when the accessed key evaluates to `None`, instead of the specified default value.
```

Comment:
I think this is a bug fix, not an API change.
Also, maybe split into two items, lumping DictConfig and ListConfig makes it hard because DictConfig motivation is to be consistent with plain dict and ListConfig motivation is to be consistent with DictConfig (plain list does not have a get method).

Reply:
Done in d2b0cf6